### DISCUSSION TITLE
Is it possible to run mikropml through GPU?

### DISCUSSION THREAD

**Discussion by sskoldas** [SchlossLab/mikropml#312] [Q&A] [answered]
Can I run mikropml through GPU in R environment. If I can which steps should I follow? I am a newbie in this field.

**Comment by kelly-sovacool** [comment, marked as answer]
We use the `caret` package which doesn't support any GPU backend for parallel processing, as far as I'm aware of. See https://github.com/topepo/caret/issues/391

You should be able to get reasonably fast run times with multicore processing on an HPC cluster. We have an example Snakemake workflow which is great for running jobs in parallel on HPCs: https://github.com/SchlossLab/mikropml-snakemake-workflow
